Add changes to oidc docs in concepts section. (#2636)
wso2/docs-is: en/docs/references/concepts/authentication/basic-client-profile.md

These calls are to:

1. Obtain the one-time password by calling the authorization endpoint and sharing the user credentials. This flow uses the front channel<!--[front channel](insertlink)-->. 

2. Exchange the one-time password with an access token<!--[access token](insertlink)--> by calling the token endpoint. This flow requires (back channel)
client authentication using 'client id' and 'client secret'. 

This approach is recommended for confidential clients<!--[confidential clients](insertlink)--> 
who can maintain the confidentiality of their credentials (e.g., web apps). As the tokens are not visible via the user 

> agents, this flow is more secure and allows for refresh tokens<!--[refresh tokens](insertlink)-->, which guarantees long-lived access.

Reviewer: ```suggestion
agents, this flow is more secure and allows to refresh tokens<!--[refresh tokens](insertlink)--> guaranteeing long-lived access.
```
